nemo/collections/asr/parts/mixins/mixins.py

```diff
@@ -83,7 +83,7 @@ def _setup_monolingual_tokenizer(self, tokenizer_cfg: DictConfig):
                 with open_dict(self.cfg.tokenizer):
                     self.cfg.tokenizer.hf_kwargs = tokenizer_cfg.get('hf_kwargs')
 
-        if self.tokenizer_type not in ['bpe', 'wpe']:
+        if self.tokenizer_type not in ['bpe', 'wpe', 'yttm']:
```

Comment:
Dont support yttm its mostly deprecated. https://github.com/NVIDIA/NeMo/issues/6854

Reply:
Sentencepiece with spe mode does just the same thing as yttm